src/frontend/App/context/AppContext.tsx

```diff
@@ -62,17 +63,17 @@ const [AppProvider, useApp] = constate(({ autentisertSaksbehandler, appEnv }: IP
         }
     };
 
-    const axiosRequest = useCallback(
-        <T, D>(
-            config: AxiosRequestConfig & { data?: D }
-        ): Promise<RessursFeilet | RessursSuksess<T>> => {
+    const axiosRequest: AxiosRequestCallback = useCallback(
+        <RES, REQ>(
+            config: AxiosRequestConfig<REQ>
```

Comment:
`AxiosRequestConfig & { data?: D }` trengs ikke då `AxiosRequestConfig<REQ>` allerede setter det samme. Litt fortydligende her for å skjønne hva `T` og `D` er

Reply:
Du mener at det går an å unngå å skjønne forskjellen på T og D ? 😂 🤯 😕